src/mono/mono/mini/type-checking.c

```diff
@@ -40,6 +40,7 @@ emit_cached_check_args (MonoCompile *cfg, MonoInst *obj, MonoClass *klass, int c
 	} else {
 		int idx;
 
+		g_assert (klass);
```

Comment:
Even though this really looks is correct, from a conservative perspective I wouldn't add any additional asserts to backport PRs.

Reply:
@kg the easiest option is actually to use the github UI and make a suggestion to yourself to remove the line. like this:
```suggestion
```